Resources/config/services.yml

```diff
@@ -64,3 +69,10 @@ services:
             - "%kernel.bundles%"
         tags:
             - { name: "support_tools.system_info.collector", identifier: "symfony_kernel" }
+
+    # SystemInfoOutputFormats
+
+    support_tools.system_info.output_format.json:
+        class: "%support_tools.system_info.output_format.json.class%"
+        tags:
+            - { name: "support_tools.system_info.output_format", identifier: "json" }
```

Comment:
Sounds good. Note the concrete implementation of InfoCollectorRegistry interface is "IdentifierBased" so in case the tag changes a minor refactoring/renaming would be needed for consistency (also for vars etc), but not a big deal imo.